buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -236,16 +236,6 @@ object Libs {
    */
   const val objectbox_windows: String = "io.objectbox:objectbox-windows:" + Versions.io_objectbox
 
-  /**
-   * http://jacoco.org
-   */
-  const val org_jacoco_agent: String = "org.jacoco:org.jacoco.agent:" + Versions.org_jacoco
```

Comment:
Why we are removing these dependencies?

Reply:
These dependencies are not removed. These lines are being removed because they are misleading and would make the reader want to believe that Gradle is reading the versions from here. Gradle is not. A `grep -r org_jacoco` returns nothing, and this is the same for the objectbox line above (which is another topic so I can't change in this PR). Only the lines returned by `grep -r 'Libs\.' --include '*.kt'` are actually being used.

The actual version is set in `lintConfig.xml`, `jacoco.gradle`, and `buildSrc/src/main/kotlin/plugin/AllProjectConfigurer.kt`. Changing the versions in `Libs.kt` did nothing, and an XML file can't import the values from a Kotlin file.

Reply:
Good catch! But be sure to put this change in a specific commit with a commit message explicating why we do this change. it'll easy to recognise why we done this. Because this way it's hard to recognise this change.

Reply:
I will move it to a separate PR where I deal with all of the similar lines in that file. It would make sense that a separate commit deal with the whole topic of unused variable all at once.